types/react-table/index.d.ts

```diff
@@ -333,25 +338,29 @@ export type UseExpandedOptions<D extends object> = Partial<{
 }>;
 
 export interface UseExpandedHooks<D extends object> {
-    getExpandedToggleProps: Array<(row: Row<D>, instance: TableInstance<D>) => object>;
+    getToggleAllRowsExpandedProps: Array<(row: Row<D>, instance: TableInstance<D>) => object>;
```

Comment:
Hmmm shouldn't this be `getToggleRowExpandedProps`?

> Renamed the row.getExpandedToggleProps to row.getToggleRowExpandedProps

Reply:
Depending upon that resolution, may need to add `getToggleAllRowsExpandedProps`

> Added the instance.getToggleAllRowsExpandedProps prop getter

Reply:
I think it's both, isn't it? `getToggleRowExpandedProps` AND `getToggleAllRowsExpandedProps`?

Reply:
```suggestion
    getToggleAllRowsExpandedProps: Array<(row: Row<D>, instance: TableInstance<D>) => object>;
    getToggleRowExpandedProps: Array<(row: Row<D>, instance: TableInstance<D>) => object>;
```

Reply:
The whole *Hooks set of interfaces is a bit of a mess since most of the methods actually end out on UseTableHooks, looking at them I think that most of them are out of date and wrong.  The are only used inside plugins so it's easy to miss.

Reply:
Yeah, I agree. It's kinda hard to tell how all of those really should be implemented. FWIW, this works with my implementation of `getToggleRowExpandedProps()` and other new entities. Hoping it goes in soon! ;)